StandaloneMmPkg/Include/Ppi/MmCoreFvLocationPpi.h

```diff
@@ -0,0 +1,35 @@
+/** @file
+  MM Core FV location PPI header file.
+
+  MM Core FV location PPI is used by StandaloneMm IPL to find MM Core
+  .

```

Comment:
MM Core FV location PPI is used by StandaloneMm IPL to find MM Core.

"." is in another line

Reply:
updated